docs/Bigtable-batchsource.md

```diff
@@ -31,6 +31,8 @@ It can be found on the Dashboard in the Google Cloud Platform Console.
 
 **Key Alias**: Name of the field for row key.
 
+**Column Mappings**: Mappings from Bigtable column name to record field. Column name format: <family>:<qualifier>.
```

Comment:
As I said on last weekly call, I think it could be confusing for user to have both columnFamily and mappings without some select on UI that will affect what properties to set. For some reason I understood your answer as keep mappings for now until UI will support such conditional properties. So should this be delayed or is it better to have both options now?

Reply:
the UI is currently building this out (design at https://wiki.cask.co/display/CE/Dynamic+plugin+property+filters+for+CDAP+Pipelines).

We can revisit this in a future PR once it's been added and only focus on the column mapping for now. Please open a jira so it's not lost.

Reply:
created Jira issue https://issues.cask.co/browse/CDAP-15662